### PR TITLE
Support Incidents API

### PR DESCRIPTION
**Pull request by GeneticGenesis**
* Support Incidents API.
* Bump minor version.

### CODE REVIEW

File: api_incidents.go [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+		localVarReturnValue  IncidentResponse
+	)
+
+	for _, opt := range opts {
+		opt(localVarAPIOptions)
+	}
+
+	// create path and map variables
+	localVarPath := a.client.cfg.basePath + "/data/v1/incidents/{INCIDENT_ID}"
+	localVarPath = strings.Replace(localVarPath, "{"+"INCIDENT_ID"+"}", fmt.Sprintf("%v", iNCIDENTID), -1)
```
Comment (jsanford8) [inline api_incidents.go]: What the... I see the signature is `iNCIDENTID`, but that just feels really weird. If this is go, chalk it up to another "I'm not sure what Go is doing", but just thought I'd ask.

File: docs/Incident.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+**MeasuredValue** | **float64** |  | [optional] 
+**IncidentKey** | **string** |  | [optional] 
+**Impact** | **string** |  | [optional] 
+**Id** | **string** |  | [optional] 
+**ErrorDescription** | **string** |  | [optional] 
+**Description** | **string** |  | [optional] 
+**Breakdowns** | [**[]IncidentBreakdown**](IncidentBreakdown.md) |  | [optional] 
+**AffectedViewsPerHourOnOpen** | **float64** |  | [optional] 
+**AffectedViewsPerHour** | **float64** |  | [optional] 
+**AffectedViews** | **float64** |  | [optional]
```
Comment (jsanford8) [inline docs/Incident.md]: Same as PHP, so I'm thinking this may just be openAPI, but this should actually be an integer.

File: docs/Incident.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+## Properties
+Name | Type | Description | Notes
+------------ | ------------- | ------------- | -------------
+**Threshold** | **float64** |  | [optional] 
+**Status** | **string** |  | [optional] 
+**StartedAt** | **string** |  | [optional] 
+**Severity** | **string** |  | [optional] 
+**SampleSizeUnit** | **string** |  | [optional] 
+**SampleSize** | **float64** |  | [optional]
```
Comment (jsanford8) [inline docs/Incident.md]: int64?

File: docs/IncidentsApi.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+Get an Incident
+
+Returns the details of an incident 
+
+### Required Parameters
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **ctx** | **context.Context** | context for authentication, logging, cancellation, deadlines, tracing, etc.
+  **iNCIDENTID** | **string**| ID of the Incident |
```
Comment (jsanford8) [inline docs/IncidentsApi.md]: Here we are again with the `iNCIDENT`... I hope this doesn't show up in the docs.

File: docs/IncidentsApi.md [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+ **optional** | ***ListIncidentsOpts** | optional parameters | nil if no parameters
+
+### Optional Parameters
+Optional parameters are passed through a pointer to a ListIncidentsOpts struct
+
+Name | Type | Description  | Notes
+------------- | ------------- | ------------- | -------------
+ **limit** | **optional.Int32**| Number of items to include in the response | [default to 25]
+ **page** | **optional.Int32**| Offset by this many pages, of the size of &#x60;limit&#x60; | [default to 1]
+ **orderBy** | **optional.String**| Value to order the results by |
```
Comment (jsanford8) [inline docs/IncidentsApi.md]: No listing of possible values (I feel like I'm repeating myself across PRs... sorry)

**Comment by GeneticGenesis** [comment]
We've backed out incidents support for now, we'll pick this up again soon.

**Pull request closed without merging**